__tests__/tokens_utils.js

```diff
@@ -138,6 +138,8 @@ test('New token', async (done) => {
   const ret = tokens.createToken(address, tokenName, tokenSymbol, 200, pin);
   const sendTransaction = ret.sendTransaction;
   sendTransaction.start();
+  // We must do this, otherwise the setTimeout method will be called with a null localStorage
+  sendTransaction.updateOutputSelected(false);
```

Comment:
Is it related to the different versions of nodejs?

Reply:
It's not. Without this the test was not failing but an async exception is raised because of a `setTimeout` that is called after the test is finished. This prevents this `setTimeout` from being called.

Reply:
Ok. Thanks for the clarification.